test/server/core/abilities/keyword/Sentinel.spec.js

```diff
@@ -0,0 +1,51 @@
+describe('Sentinel keyword', function() {
+    integration(function() {
+        describe('When a unit with the Sentinel keyword', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['liberated-slaves'],
+                    },
+                    player2: {
```

Comment:
Can we add some additional non-sentinel units in both of these test cases just to be thorough